recipes/bacnet-stack/all/test_package/conanfile.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class TestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def imports(self):
```

Comment:
Copying these libraries should not be necessary as conan will handle the dependencies.
```suggestion
```